### PR TITLE
Add Fetch and Move PG AST nodes

### PR DESCRIPTION
**Pull request by mulias**
Add https://www.postgresql.org/docs/current/sql-fetch.html and https://www.postgresql.org/docs/current/sql-move.html which are related

### CODE REVIEW

File: packages/sij-dialect-postgresql/src/ast/util.ts [hunk 1]
```diff
@@ -13,6 +17,83 @@ type PgUtil =
...
+
+type DirectionOption =
+  | 'Next'
+  | 'Prior'
+  | 'First'
+  | 'Last'
+  | Absolute
+  | Relative
+  | 'All'
+  | 'Forward'
```
Comment (ClaireNeveu) [inline packages/sij-dialect-postgresql/src/ast/util.ts:54]: I would make these duplicate cases into one case with the `count` as `NumLit | null`.

**Review by ClaireNeveu — CHANGES_REQUESTED** [review]

File: packages/sij-dialect-postgresql/src/ast/util.ts (hunk 1 above)
Comment (mulias) [reply packages/sij-dialect-postgresql/src/ast/util.ts:54]: To be more general, would you prefer if this list had the minimal number of options necessary to cover all cases? For example `Next`,  and `Forward`, are equivalent to `Forward 1` or `Relative 1`. I think I could get rid of `Next`, `Prior`, `First`, `Last`,  `All`, `Forward`, and `Backward`. That would leave `Absolute count`, `Relative count`, `Forward count`, `Backward count`, `ForwardAll`, and `BackwardAll`.

Comment (mulias) [reply packages/sij-dialect-postgresql/src/ast/util.ts:54]: It's not clear to me which you're seeing as the duplicate cases.

Comment (mulias) [reply packages/sij-dialect-postgresql/src/ast/util.ts:54]: Oh I see it, we could actually do `count: NumLit | 'All' | null`

**Review by ClaireNeveu — APPROVED** [review]

**Pull request merged**